sound/soc/sof/topology.c

```diff
@@ -1719,6 +1719,85 @@ static int sof_widget_load_siggen(struct snd_soc_component *scomp, int index,
 	return ret;
 }
 
+static int sof_get_control_data(struct snd_sof_dev *sdev,
```

Comment:
@juimonen general comment on this comment:
On one hand I appreciate the logic of trying to make the change then refactor
On the other hand, this looks like tick-tock patches where you change what you just introduced.
Maybe make sure that this a block change and avoid additional changes such as changed comments or changed sequences, etc?

Reply:
@plbossart I tried to remove my changes to all non-related comments even they would be originally erroneous. That can be addressed in another pr.